content/building/automatic-build-triggering.md

```diff
@@ -48,11 +48,35 @@ To set up a webhook for automatic build triggering on Codemagic, you must regist
 
 {{% notebox %}}
 The payload URL has the following format:  
-`https://api.codemagic.io/hooks/[yourappID]`
+`https://api.codemagic.io/hooks/[appId]`
 {{% /notebox %}}
 
 You can find your app ID in the browser URL after `app/` when you open the app on Codemagic: `https://codemagic.io/app/[yourappID]`
 
+## Custom build triggers
+
+Build triggering in response to custom events can be set up using [AWS Lambda](https://aws.amazon.com/lambda/). In AWS Lambda function, call the following command: 
```

Comment:
without AWS Lambda, just sending POST request to `https://api.codemagic.io/builds` endpoint